controller/engine_controller.go

```diff
@@ -1657,6 +1657,11 @@ func (ec *EngineController) startRebuilding(e *longhorn.Engine, replica, addr st
 		return nil
 	}
 
+	replicaObj, err := ec.ds.GetReplica(replica)
```

Comment:
Can we rename replica to replicaName and replace replicaObj with replica? Thanks.

Reply:
Renamed. Thanks